utils/build-script-impl

```diff
@@ -2516,6 +2520,40 @@ for host in "${ALL_HOSTS[@]}"; do
                     LIBICU_BUILD_ARGS=()
                 fi
 
+                if [[ ! "${SKIP_BUILD_LIBXML2}" ]]; then
+                    BASE_INSTALL_DIR="$(get_host_install_destdir ${host})"
+                    LIBXML2_HEADERS="${BASE_INSTALL_DIR}/usr/include/libxml2"
+                    LIBXML2_BUILD_ARGS=(-DLIBXML2_INCLUDE_DIR:PATH="$LIBXML2_HEADERS")
+                    # Find static library
+                    LIBXML2_LIB=$(find "${BASE_INSTALL_DIR}"  -name 'libxml2.a')
+                    if [[ ! "${LIBXML2_LIB}" ]]; then
+                        LIBXML2_LIB=$(find "${BASE_INSTALL_DIR}" -name 'libxml2.so' -o -name 'libxml2.dll')
```

Comment:
`find` won't work for Windows, so the `.dll` is meaningless.  Additionally, the static library name is incorrect for windows.